src/components/MyProfile/MyGoal.tsx

```diff
@@ -0,0 +1,162 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { IcAdd, IcMinusWhite } from '../../assets';
+
+const MyGoal = () => {
+  const [number, setNumber] = useState(0);
+  const [isSaved, setIsSaved] = useState(false);
+
+  const handleSaveBtnClick = () => {
+    setIsSaved(true);
+  };
+
+  const handleCancelBtnClick = () => {
+    setIsSaved(false);
+  };
+
+  const isActionAllowed = () => !isSaved;
+
+  // 증가 함수: isActionAllowed가 true일 경우에만 작동
+  const increaseNumber = () => {
+    if (isActionAllowed() && number < 7) {
+      setNumber((prev) => prev + 1);
+    }
+  };
```

Comment:
위의 리뷰처럼 수정한 경우 아래와 같이 구현 가능
```suggestion
  const increaseNumber = () => {
    if (isActionAllowed && number < 7) {
      setNumber((prev) => prev + 1);
    }
  };
```

Reply:
오 그렇군용! 수정하겠습니당